### PR TITLE
fix: expose the update_password component as a view

### LINKED ISSUE #193: Feat: Show change password view on <Auth/> component
I’m trying to work out a flow for a forgotten password in my application, but the Auth component’s change password view doesn’t seem to be linked up to anything.

### PR DESCRIPTION
**Pull request by jin-exists**
fixes #193 

Just exposes the `update_password` component in the UI Auth in the view map so it can be conditionally shown with the view prop.

### CODE REVIEW

File: src/components/Auth/Auth.tsx [hunk 1]
```diff
@@ -123,6 +124,15 @@ function Auth({
         </Container>
       )
       break
+    case VIEWS.UPDATE_PASSWORD:
+      return (
+        <Container>
+          <UpdatePassword
+            supabaseClient={supabaseClient}
+            setAuthView={setAuthView}
```
Comment (YashKandalkar) [inline src/components/Auth/Auth.tsx:132]: The UpdatePassword component doesn't seem to have setAuthView prop

**Review by YashKandalkar — CHANGES_REQUESTED** [review]

**Comment by YashKandalkar** [comment]
You might also want to add `'update_password'` in [Auth.stories.tsx](https://github.com/supabase/ui/blob/develop/src/components/Auth/Auth.stories.tsx) since it's not there

File: src/components/Auth/Auth.tsx (hunk 1 above)
Comment (MildTomato) [reply src/components/Auth/Auth.tsx:132]: @jin-exists you ok to fix this?

Comment (jin-exists) [reply src/components/Auth/Auth.tsx:132]: The component does't use setAuthView at all so I'm not sure whether adding it is necessary?

**Comment by jin-exists** [comment]
> You might also want to add `'update_password'` in [Auth.stories.tsx](https://github.com/supabase/ui/blob/develop/src/components/Auth/Auth.stories.tsx) since it's not there

I would but I haven't used Storybook before and I don't want to goof anything up.

File: src/components/Auth/Auth.tsx (hunk 1 above)
Comment (MildTomato) [reply src/components/Auth/Auth.tsx:132]: oh yea, sorry !

Comment (YashKandalkar) [reply src/components/Auth/Auth.tsx:132]: I was asking to _remove_ the `setAuthView` prop btw, which you accidentally gave to the UpdatePassword component. 
I didn't make the proper changes I think so it looked like I've added it 😅

Comment (MildTomato) [reply src/components/Auth/Auth.tsx:132]: Sorry everyone! I'm confusing everyone.
setAuthView could be removed, I can just make some changes on the develop branch :D

**Pull request merged**